llvm/test/ExecutionEngine/JITLink/x86-64/ELF_vtune.s

```diff
@@ -0,0 +1,52 @@
+# REQUIRES: native && x86_64-linux && intel-jitevents
+
+# RUN: rm -rf %t && mkdir -p %t
```

Comment:
remove `-p` since the file/directory is guaranteed to be absent

Reply:
Done!